tests/test_apis.py

```
@@ -74,16 +76,49 @@
-                    assert (
-                        value.items() <= response_dict[key].items()
-                    ), f"Expected {key} to contain {value}, but got {response_dict.get(key)}"
+                    try:
+                        assert value.items() <= response_dict[key].items()
+                    except AssertionError:
+                        error_detail = find_first_error(
+                            value, response_dict[key]
+                        )
+                        raise AssertionError(
+                            f"Expected {key} to contain {value}, but got {response_dict.get(key)}. \n\nDetail: {error_detail}"
+                        )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)